services/src/test/java/org/fao/geonet/services/region/MetadataRegionDAOTest.java

```diff
@@ -99,6 +100,7 @@ public void setUp() throws Exception {
     }
 
     @Test
+    @Ignore(value = "Not running anymore")
```

Comment:
I would say GeoNetwork user interface is not using MetadataRegion (@josegar74 you confirm?). I think this comes from geocat.ch and even in current geocat.ch I think that it is not used anymore too but that is something to check. So this is maybe an unused feature to simply remove. BTW, the error is related to GML3.2 geom extraction and parsing of geom.